addons/hr_holidays/__manifest__.py

```diff
@@ -47,6 +47,7 @@
         'report/hr_leave_reports.xml',
 
         'views/hr_views.xml',
+        'views/hr_leave_template.xml',
```

Comment:
hr_time_off_template.xml ?

Reply:
@legalsylvain Thanks for the feedback. 

According to guidelines, we use the model technical name, not the user one.